src/visual.jl

```diff
@@ -60,6 +60,9 @@ end
 function Base.show(io::IO, ::MIME"text/plain", obj::SmallObject)
     print(io, show_small_obj(obj))
 end
+function Base.show(io::IOContext, ::MIME"text/plain", obj::SmallObject)
```

Comment:
Why `IOContext` method? `IOContext <: IO` ?

Reply:
this method was neccessary to avoid disambiguities. `SparseArrays` module also has a method `show` which is more precise in the `IO` part, whereas our function is more precise for `Polymake.SparseVector` in the `AbstractSparseArray` part:

```julia
julia> methodswith(AbstractSparseVector,show)
[1] show(io::IO, ::MIME{Symbol("text/plain")}, x::AbstractSparseArray{Tv,Ti,1} where Ti where Tv) in SparseArrays at /buildworker/worker/packa
ge_linux64/build/usr/share/julia/stdlib/v1.3/SparseArrays/src/sparsevector.jl:836
[2] show(io::IOContext, x::AbstractSparseArray{Tv,Ti,1} where Ti where Tv) in SparseArrays at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/SparseArrays/src/sparsevector.jl:848
[3] show(io::IO, x::AbstractSparseArray{Tv,Ti,1} where Ti where Tv) in SparseArrays at /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.3/SparseArrays/src/sparsevector.jl:845
```